src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -1030,6 +1035,37 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
             'value': frontend
         })
 
+        if spec.rgw_user_counters_cache:
+            ret, out, err = self.mgr.check_mon_command({
+                'prefix': 'config set',
+                'who': utils.name_to_config_section(daemon_spec.name()),
+                'name': 'rgw_user_counters_cache',
+                'value': spec.rgw_user_counters_cache_size,
+            })
+        if spec.rgw_bucket_counters_cache:
+            ret, out, err = self.mgr.check_mon_command({
+                'prefix': 'config set',
+                'who': utils.name_to_config_section(daemon_spec.name()),
+                'name': 'rgw_bucket_counters_cache',
+                'value': spec.rgw_bucket_counters_cache_size,
```

Comment:
typos? the `value` fields have `_size` at the end

Reply:
Yes, fixed it. Thanks for noticing!